File: Python/programmers/etc/level1/음양_더하기.py
def solution(absolutes, signs):
    answer = 0
    for i in range(len(absolutes)):
        if signs[i]:
            answer += absolutes[i]
        else:
            answer -= absolutes[i]
    return answer


absolutes = [4,7,12]
signs = [True,False,True]
print(solution(absolutes, signs)) # 9

absolutes = [1,2,3]
signs = [False,False,True]
print(solution(absolutes, signs)) # 0

File: Java/programmers/etc/level1/음양더하기.java
package CodeTest.Java.programmers.etc.level1;

public class 음양더하기 {
    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;

        for (int i=0; i<absolutes.length; i++) {
            answer += absolutes[i] * (signs[i] ? 1: -1);
        }

        return answer;
    }
}
